src/app/clients/getClientsTableData.ts

```diff
@@ -1,20 +1,25 @@
 import supabase from "@/supabaseClient";
 import { Schema } from "@/database_utils";
-import { Datum } from "@/components/Tables/Table";
 
-export interface ClientsTableRow extends Datum {
-    primaryKey: string;
+export interface ClientsTableRow {
     parcelId: Schema["parcels"]["primary_key"];
-    flaggedForAttention: boolean;
-    requiresFollowUpPhoneCall: boolean;
+    primaryKey: Schema["clients"]["primary_key"];
     fullName: Schema["clients"]["full_name"];
     familyCategory: string;
     addressPostcode: Schema["clients"]["address_postcode"];
-    collectionCentre: string;
-    congestionChargeApplies: boolean;
+    deliveryCollection: {
+        collectionCentre: string;
+        congestionChargeApplies: boolean;
+    };
     packingTimeLabel: string;
-    collectionDatetime: string;
+    collectionDatetime: Schema["parcels"]["collection_datetime"];
     lastStatus: string;
+    voucherNumber: Schema["parcels"]["voucher_number"];
+    iconsColumn: {
+        flaggedForAttention: boolean;
```

Comment:
Should this be `Schema["clients"]["flagged_for_attention"]`

Reply:
Although it is just a boolean - but not sure if it is more restrictive in terms of the meaning behind it

Reply:
I wanted to decouple what's in the UI from the database schema, so if the schema ever changed it would raise a type error at the point at which we want the error to occur